spec/system/candidate_interface/continuous_applications/submitting/candidate_sees_all_their_applications_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
```

Comment:
This file is in the submitting directory but no applications are being submitted. I think we should put this in a different directory

Reply:
Good suggestion. I move to `spec/system/candidate_interface/continuous_applications/viewing-application-choices`.